cvat/apps/lambda_manager/views.py

```diff
@@ -725,12 +747,32 @@ def retrieve(self, request, func_id):
         gateway = LambdaGateway()
         return gateway.get(func_id).to_dict()
 
+    @extend_schema(description=textwrap.dedent("""\
+        Allows to execute a function for immediate computation.
+
+        Supposed for short-living executions, useful for interactive calls.
+        When executed for interactive annotation, the job id must be specified
+        in the 'job' input field.
+        """),
+        request=inline_serializer("OnlineFunctionCall", fields={
+            "job": serializers.IntegerField(required=False),
+            "task": serializers.IntegerField(required=False),
+        }),
+        responses=OpenApiResponse(description="Returns function invocation results")
+    )
     @return_response()
     def call(self, request, func_id):
         self.check_object_permissions(request, func_id)
         try:
-            task_id = request.data['task']
-            db_task = TaskModel.objects.get(pk=task_id)
+            job_id = request.data.get('job')
+            job = None
+            if job_id is not None:
+                job = Job.objects.get(id=job_id)
+                task_id = job.get_task_id()
```

Comment:
IMO, in this case you should check that the `task_id` in the query is unset (or set to the job's task ID). Otherwise, the endpoint will behave confusingly if a user accidentally makes a query with mismatched job and task IDs.